rln/utils.go

```diff
@@ -56,6 +75,14 @@ func Bytes128(b []byte) [128]byte {
 	return result
 }
 
+func Flatten(b [][32]byte) []byte {
+	var result []byte
+	for _, v := range b {
+		result = append(result, v[:]...)
+	}
```

Comment:
```suggestion
	result := make([]byte, len(b))
	for i, v := range b {
		result[i] = v[:]...
	}
```
to avoid resizing the array

Reply:
fixed here 8190b2044bd63439d72f3fce4e79fd38a8195753
slightly different though.

but yeap, better to allocate the array deterministically, since appending in each iteration is a waste of resources.

quick measurements with Go benchmark.

old flatten
41.43 ns/op	      96 B/op	       2 allocs/op

new flatten
21.59 ns/op	      64 B/op	       1 allocs/op


thanks!